docs/tutorial/part-four/index.md

```diff
@@ -66,7 +66,7 @@ to you.
 
 Gatsby uses GraphQL to enable components to declare the data they need.
 
-## Your first GraphQL query
+## Create a new example site
```

Comment:
Changing this will break the "Your first GraphQL query" link in the sidebar:

<img width="327" alt="screen shot 2018-08-10 at 18 52 08" src="https://user-images.githubusercontent.com/381801/43973132-86dc5524-9cce-11e8-9922-72f6f4bd2908.png">

I think "Your first GraphQL query" works well for the sidebar, so maybe it makes sense to keep it here? Or change it to something else. Anyway I think this is good to merge, and the sidebar can be updated in a follow-up PR.